OpenSim/Simulation/Manager/Manager.h

```diff
@@ -180,9 +180,19 @@ class OSIMSIMULATION_API Manager
     void setIntegrator(SimTK::Integrator&);
 
     // Initial and final times
+    /** <b>(Deprecated)</b> Set the state's time using state.setTime(time). */
```

Comment:
You can get doxygen to link to the correct SimTK method if you write something like `SimTK::State::getTime()`.